---
layout: paper
title: "Quantifying codon usage in signal peptides: Gene expression and amino acid usage explain apparent selection for inefficient codons"
image: 
authors: Cope AL, Hettich RL, Gilchrist MA.
year: 2018
ref: Cope et al. 2018. Biochimica et Biophysica Acta (BBA) - Biomembranes.
journal: "Biochimica et Biophysica Acta (BBA) - Biomembranes 1860(12): 2479-2485"
pdf: 
doi: https://doi.org/10.1016/j.bbamem.2018.09.010
tags: CUB
---

# Abstract

The Sec secretion pathway is found across all domains of life. A critical feature of Sec secreted proteins is the signal peptide, a short peptide with distinct physicochemical properties located at the N-terminus of the protein. Previous work indicates signal peptides are biased towards translationally inefficient codons, which is hypothesized to be an adaptation driven by selection to improve the efficacy and efficiency of the protein secretion mechanisms. We investigate codon usage in the signal peptides of E. coli using the Codon Adaptation Index (CAI), the tRNA Adaptation Index (tAI), and the ribosomal overhead cost formulation of the stochastic evolutionary model of protein production rates (ROC-SEMPPR). Comparisons between signal peptides and 5′-end of cytoplasmic proteins using CAI and tAI are consistent with a preference for inefficient codons in signal peptides. Simulations reveal these differences are due to amino acid usage and gene expression – we find these differences disappear when accounting for both factors. In contrast, ROC-SEMPPR, a mechanistic population genetics model capable of separating the effects of selection and mutation bias, shows codon usage bias (CUB) of the signal peptides is indistinguishable from the 5′-ends of cytoplasmic proteins. Additionally, we find CUB at the 5′-ends is weaker than later segments of the gene. Results illustrate the value in using models grounded in population genetics to interpret genetic data. We show failure to account for mutation bias and the effects of gene expression on the efficacy of selection against translation inefficiency can lead to a misinterpretation of codon usage patterns.
