---
layout: paper
title: Exploring ribosome-positioning on translating transcripts with ribosome profiling
authors: Cope AL, Vellappan S, Favate JS, Skalenko KS, Yadavalli SS, Shah P.
year: 2021
ref: Cope et al. 2021. Methods in Molecular Biology. 
journal: "Methods in Molecular Biology 2404: 83-110 "
pdf: 
doi: https://doi.org/10.1007/978-1-0716-1851-6_5
---

# Abstract 

The emergence of ribosome profiling as a tool for measuring the translatome has provided researchers with valuable insights into the post-transcriptional regulation of gene expression. Despite the biological insights and technical improvements made since the technique was initially described by (Ingolia et al., 2009), ribosome profiling measurements and subsequent data analysis remain challenging. Here, we describe our labâ€™s protocol for performing ribosome profiling in bacteria, yeast, and mammalian cells. This protocol has integrated elements from three published ribosome profiling methods (McGlincy and Ingolia, 2017; Mohammad et al., 2019; Subtelny et al., 2014). In addition, we describe a tool called RiboViz (Carja et al., 2017) (https://github.com/riboviz/riboviz) for the analysis and visualization of ribosome profiling data. Given raw sequencing reads and transcriptome information (e.g., FASTA, GFF) for a species, RiboViz performs the necessary pre-processing and mapping of the raw sequencing reads. RiboViz also provides the user with various quality control visualizations.